scripts/js/upgrade/index.js

```diff
@@ -4,16 +4,24 @@ const fs = require('fs')
 const util = require('util');
 const exec = util.promisify(require('child_process').exec);
 
+const PREIMAGE_LENGTH_BOUND = 34;
```

Comment:
How are these values obtained or updatable based on what in the future?

Reply:
I.e, are these values always the same for any pre-image proposing authorising a runtime upgrade? Or do we need to keep changing them? If so, what's the best way to do that? I am 90% we don't, but just want to double check.